docs/docs/makefile.md

```diff
@@ -1,8 +1,10 @@
+## Windows
 In order to build an MTA archive from a project's source code in a Windows environment, you should install `GNU Make 4.2.1` on your machine.
```

Comment:
To build an MTA archive from a project's source code in a Windows environment, install `GNU Make 4.2.1` on your machine.

Reply:
Done